### PR TITLE
chore: Add workflow trigger

### PR DESCRIPTION
**Pull request by ryjones**
This will let people trigger builds without a change.

Signed-off-by: Ry Jones <ry@linux.com>

### CODE REVIEW

**Comment by WadeBarnes** [comment]
(ci) test this please.

**Review by WadeBarnes — APPROVED** [review]

**Comment by WadeBarnes** [comment]
(ci) test this please.

**Comment by WadeBarnes** [comment]
@Toktar, @skhoroshavin, We just need an approval from a code owner here.

**Review by Toktar — APPROVED** [review]

**Pull request merged**
